.github/workflows/ci.yaml

```
@@ -26,7 +31,12 @@ jobs:
       - run: cp .github/workflows/kernel-${{ matrix.mode }}.config .config
       - if: matrix.module == 'loadable'
         run: sed -i -E 's/^(CONFIG_RUST_EXAMPLE=)(y)$/\1m/g' .config
-      - run: make CC=clang-10 LLVM_CONFIG_PATH=llvm-config-10 -j3
+      - if: matrix.outputdir == 'src'
+        run: make ${MAKE_FLAGS}
+      - if: matrix.outputdir == 'build'
+        run: mkdir ${BUILD_DIR}
+        run: mv .config ${BUILD_DIR}/.config
+        run: make ${MAKE_FLAGS} O=${BUILD_DIR}
```

```suggestion
        run:
          - mkdir ${BUILD_DIR}
          - mv .config ${BUILD_DIR}/.config
          - make ${MAKE_FLAGS} O=${BUILD_DIR}
```

I think it was like this, but not sure

---

Almost, it is the `|` to start a multiline and without `-`.

---

yeah, I remeber. thats why I like json over yaml.

Oh wait, it's the same xP